sdk/src/main/java/com/qonversion/android/sdk/Qonversion.kt

```diff
@@ -159,4 +160,22 @@ interface Qonversion {
      * for the most part of use cases.
      */
     fun finish()
+
+    /**
+     * Get the most recent available User info.
+     * @return [User] instance.
```

Comment:
Let's leave an empty line between description and parameter list in kdoc comments for more readability?